### ISSUE TITLE
Feature request: send point values

### ISSUE THREAD

**Issue by cyberic99** [jean-emmanuel/open-stage-control#130]
I don't know if it is possible, but:
If I try to set a text value, it works as intended:
```
oscsend osc.udp://localhost:8080 /SET ss  text_1  mytext
```

But for points, it doesn't seem to work...  
```
oscsend osc.udp://localhost:8080 /SET ss  plot_1  '[[1,2],[2,2]]'
```
does nothing...
In addition, is it possible to set the value of one point individually?

Thanks, and thanks once again for Open Stage Control!

**Comment by jean-emmanuel** [comment]
Hmm, the plot widget is a bit inconsistent there, it needs some tinkering, I'll get to it soon.

**Comment by cyberic99** [comment]
ok, thank you
While you are at it, do you think it might be useful to be able to draw bargraphs?

**Comment by jean-emmanuel** [comment]
Plots can receive stringified arrays now, setting one point's coordinates could be cool, it's easy to implement, but just need to figure a good string syntax.

> do you think it might be useful to be able to draw bargraphs?
do you ?

**Comment by cyberic99** [comment]
cool thx!

Maybe an /EDIT/MERGE id {plot_idx: [x,y]} would work?
It would allow to set/replace one or more points at once...
But it is not genuine OSC...

Or maybe /widget_id/plot plot_idx plot_value ?

yes I think bargraphs would be useful, to display a vu meters for instance.
For now we use a strip with multifaders ;-)
It works but I think ot is a bit overkill

**Comment by jean-emmanuel** [comment]
`bars` option shipped ! Plots can now receive stringified objects (`{}`) to change specific coordinates: sending `"{0:1, 4:0}"` will change the 1st and 5th points' coordinates to `1` and `0`.

**Comment by jean-emmanuel** [comment]
Included in v0.23, cheers !
